src/components/Button.jsx

```diff
@@ -0,0 +1,34 @@
+import { styled } from '@mui/material/styles';
+import Button from '@mui/material/Button';
+
+function BootstrapButton(props) {
```

Comment:
So we can rename this to Button
```suggestion
function Button(props) {
```